src/index.json

```diff
@@ -80,26 +80,23 @@
                     "run_requires": [
                         {
                             "requires": [
-                                "azure-cli-core",
-                                "azure-cli-iot (==0.1.11)",
-                                "azure-iothub-device-client",
-                                "azure-iothub-service-client",
-                                "azure-mgmt-iothub (==0.2.2)",
-                                "six"
+                                "msrestazure",
+                                "paho-mqtt (==1.3.1)"
                             ]
                         }
                     ],
                     "summary": "Azure IoT CLI Extension",
-                    "version": "0.1.1"
+                    "version": "0.3.1"
                 }
             }
         ],
         "image-copy-extension": [
             {
-                "filename": "image_copy_extension-0.0.4-py2.py3-none-any.whl",
-                "sha256Digest": "47d0f4293e833bb16bac1abbc63f7946dd87ac8db40378a230ec376d1f128fd3",
-                "downloadUrl": "https://files.pythonhosted.org/packages/c3/60/723ad5e968d042d9d4564af420626b661e8cde0820b6dc90f8933e6059bf/image_copy_extension-0.0.4-py2.py3-none-any.whl",
+                "filename": "image_copy_extension-0.0.5-py2.py3-none-any.whl",
+                "sha256Digest": "23d3e197bcf9d1017117c3125dd8b86f7b8357bcae831c17a9e0f45f32f9d62b",
+                "downloadUrl": "https://files.pythonhosted.org/packages/af/d0/7a2d861df60b5d93512ed9341918c614fe6a067f47d47da5063bef7326f6/image_copy_extension-0.0.5-py2.py3-none-any.whl",
```

Comment:
@tamirkamara Do you want to append to the list rather than overwrite the prev. value?
This is a list of extensions then the CLI has logic to filter the list and only install what's supported.
As this stands, once merged, a user on an older version of the CLI would not be able to install the old version of this extension.
If that's intended, it's fine to leave this as is.

Reply:
@derekbekoe Should I just add another extension with the name name but in the different version?
Also, the previous version (0.0.4) doesn't have a maxcliversion, will it automatically choose the latest compatible version?

Reply:
Can you add a new JSON object to the list that starts with `"image-copy-extension": [`?

** If I have the knack based version of the CLI**
The CLI would get the possible candidates for 'image-copy-extension' so in this case 0.0.4 and 0.0.5. Eventually the logic would choose the latest version (0.0.5) which is what we want.

** If you have the non knack based version **
You would still start of with the possible candidates but then 0.0.5 would be filtered out as it's not compatible with the current CLI version so then 0.0.4 would be installed.


source code here: https://github.com/Azure/azure-cli/blob/dev/src/command_modules/azure-cli-extension/azure/cli/command_modules/extension/_resolve.py#L59

Reply:
IF you can modify this PR with that change, please do it. It not, create a PR against knack-based branch with your change then we can resync this PR.

Reply:
@derekbekoe this PR is to merge 2 branches, and my original one #26 is already closed and merged to the knack-based. 
Not sure if I can do anything other opening a new one?

Reply:
The only option would be to create a PR against knack-based that adds back the old entry for the 0.0.4 extension then I can merge it.